### PR TITLE
[User] Language preference base functionality

### PR DESCRIPTION
**Pull request by taracampbell**
This is a feature we are working on in CCNA. I'm putting this in Loris so everyone can take a look.

We need to offer the Training Module in both French and English in CCNA, and potentially other modules as well. This would store the user's language preference in the user table (can be edited from my preferences page).

The functionality to actually use this preference is to be implemented elsewhere. The SQL patch does not populate existing users' language preference so places where this would be used should assume that no language preference indicates the default language.

### CODE REVIEW

File: SQL/Archive/17.2/2017-06-13-addUserLanguagePreference.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+CREATE TABLE `language` (
```
Comment (jstirling91) [inline SQL/Archive/17.2/2017-06-13-addUserLanguagePreference.sql]: Need to add this to the default schema

Comment (taracampbell) [inline SQL/Archive/17.2/2017-06-13-addUserLanguagePreference.sql]: Thanks @jstirling91 :) Always forget!

Comment (taracampbell) [inline SQL/Archive/17.2/2017-06-13-addUserLanguagePreference.sql]: @jstirling91 Finally getting back to my pull requests - should be in the default schema now!

**Comment by driusan** [comment]
@taracampbell  This looks reasonable to me, but it needs to be rebased..

**Comment by taracampbell** [comment]
@driusan rebased onto aces/minor

**Comment by driusan** [comment]
I was about to merge this, but there are conflicts...

**Comment by ridz1208** [comment]
@driusan I rebased it and moved the SQL file from `SQL/Archive/17.2/...` to `SQL/Archive/...`

I still think however that the my_preference page should not have the database query to get the language list, I think it should be in a library.. which one tho ?

**Comment by driusan** [comment]
I don't think the database query should stop this, since it's blocking another PR and it's how it's currently done most places in LORIS.

**Pull request merged**